saleor/discount/interface.py

```diff
@@ -88,3 +76,25 @@ def fetch_variant_rules_info(
             )
         )
     return rules_info
+
+
+def get_rule_translations(
+    promotion: "Promotion", rule: "PromotionRule", translation_language_code: str
+):
+    promotion_translations = [
+        translation
+        for translation in promotion.translations.all()
+        if translation.language_code == translation_language_code
+    ]
+    promotion_translation = (
+        promotion_translations[0] if promotion_translations else None
+    )
```

Comment:
Maybe something like:
```suggestion
    promotion_translation = promotion.translations.filter(language_code=translation_language_code).first()
```
And same for rule_translation
WDYT?

Reply:
`promotion.transaltions` are prefetched, `promotion.translations.filter` will cause extra db query